### PR TITLE
Add gin package

### PR DESCRIPTION
**Pull request by swhsiang**
# WHAT / WHY

* Add gin package. Issue #7
* Add sub-command `server`
```
daemon-ecs-scheduler server  --help
daemon-ecs-scheduler  server --port 70
```
* Remove useless dependencies. (glide.yaml)

cc @tnachen

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,17 @@ A ECS scheduler that places daemons on every node in ECS cluster
 
 `daemon-ecs-scheduler --cluster weave-ecs-demo-cluster --tasks cadvisor:2`
 
+***Beware of the mode of gin web server, the default mode is debug mode.***
```
Comment (tnachen) [inline README.md]: Why shouldn't the default be release?

File: scheduler.go [hunk 2]
```diff
@@ -125,10 +132,39 @@ func main() {
...
+		{
+			Name:  "server",
+			Usage: "start a HTTP server. Default value is 8080.",
+
+			Action: func(c *cli.Context) (err error) {
+				var port string
+				if len(c.String(port)) > 0 {
+					port = c.String("port")
+				} else {
+					port = "8080"
```
Comment (tnachen) [inline scheduler.go]: We gotta use a different default, everyone is using 8080

File: README.md (hunk 1 above)
Comment (swhsiang) [reply README.md]: I am fine with that. I add a parameter `mode`. The default value is `release`.

File: scheduler.go (hunk 2 above)
Comment (swhsiang) [reply scheduler.go]: How about 7777 ? I use `7777` as the port number in the latest commit.

**Review by tnachen — APPROVED** [review]

**Pull request merged**